docs/pages/access-controls/guides/locking.mdx

```diff
@@ -232,7 +232,7 @@ configuration file:
 </Tabs>
 
 </TabItem>
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+<TabItem scope={["Enterprise"]} label="Teleport Enterprise">
```

Comment:
I don't think we need to make this guide part of the diff. If we do, though:
- Let's remove the `scope` parameter, which is no longer used
- I would remove the tabs and just show how to create a `cluster_auth_preference` dynamic resource, simplifying the guide. In how-to guides, I don't think we need to show all possible branches, just one that works for most users.